### ISSUE TITLE
No changelog for the releases

### ISSUE THREAD

**Issue by ggardet** [parallaxsecond/parsec#278]
There is no changelog at all for the releases. It would be great to have a list of major changes when a new version is released.

**Comment by ionut-arm** [comment]
Makes sense, I'll look into doing this retroactively. I think we'll need to set up a way to track all the changes we make from now on, there must be something generic/easy to use out there.

**Comment by ionut-arm** [comment]
That was easy, just added this in my `.bashrc`:
```bash
changelog() {
        github_changelog_generator -u parallaxsecond -p $(basename `pwd`) --no-unreleased
}
```

**Comment by ionut-arm** [comment]
I've also published Github releases for [0.5.0](https://github.com/parallaxsecond/parsec/releases/tag/0.5.0) and [0.6.0](https://github.com/parallaxsecond/parsec/releases/tag/0.6.0)
